guides/gatherers.md

```diff
@@ -242,6 +243,44 @@ Example output (in Rhai):
 #{ nodeid: 2, ring0_addr: "10.80.1.12" };
 ```
 
+### dir_scan
+
+**Argument required**: Yes
+
+This gatherer allows to scan directories due to a glob pattern provided as argument.
```

Comment:
I would change `... directories due to a` by `directories with a glob pattern`

Reply:
done!